apps/expo/src/i18n/locales/en/messages.po

```diff
@@ -0,0 +1,1732 @@
+msgid "{0} feed for #{decodedTag} is empty"
+msgstr ""
+
+#: src/app/(tabs)/(feeds,search,notifications,self)/search/index.tsx:290
+msgid "{0} posts"
+msgstr ""
+
+#: src/app/(tabs)/(feeds,search,notifications,self)/search/index.tsx:286
+msgid "#{0}"
+msgstr ""
+
+#: src/components/screens/profile/profile-info.tsx:598
+msgid "<0>{0}</0> Followers"
+msgstr ""
+
+#: src/components/screens/profile/profile-info.tsx:606
+msgid "<0>{0}</0> Following"
+msgstr ""
+
+#: src/components/screens/profile/profile-info.tsx:614
+msgid "<0>{0}</0> Posts"
+msgstr ""
+
+#: src/app/(auth)/sign-up.tsx:160
+msgid "1 of 3"
+msgstr ""
+
+#: src/app/(auth)/sign-up.tsx:273
+msgid "2 of 3"
+msgstr ""
+
+#: src/app/(auth)/sign-up.tsx:376
+msgid "3 of 3"
+msgstr ""
+
+#: src/app/(auth)/sign-up.tsx:341
+msgid "A code has been sent to {phone}. <0>Change phone number.</0>"
+msgstr ""
+
+#: src/app/settings/feed.tsx:121
+msgid "A specific feed"
+msgstr ""
+
+#: src/app/settings/_layout.tsx:117
+msgid "About"
+msgstr ""
+
+#: src/app/settings/index.tsx:85
+msgid "About Graysky"
+msgstr ""
+
+#: src/app/settings/pro.tsx:52
+msgid "Accent colour"
+msgstr ""
+
+#: src/app/settings/app.tsx:96
+msgid "Accessibility"
+msgstr ""
+
+#: src/app/settings/index.tsx:43
+msgid "Account"
+msgstr ""
+
+#: src/app/settings/account/delete-account.tsx:226
+msgid "Account deleted. Goodbye!"
+msgstr ""
+
+#: src/app/settings/account/index.tsx:28
+msgid "Account Details"
+msgstr ""
+
+#: src/app/settings/_layout.tsx:63
+msgid "Account Settings"
+msgstr ""
+
+#: src/app/add-to-list/[handle].tsx:124
+msgid "Add {0} to List"
+msgstr ""
+
+#: src/app/composer/index.tsx:681
+msgid "Add a content warning"
+msgstr ""
+
+#: src/lib/composer/alt-text-editor.tsx:154
+msgid "Add a description to the image. Good alt text is concise yet detailed. Make sure to write out any text in the image itself."
+msgstr ""
+
+#: src/components/switch-accounts.tsx:177
+msgid "Add another account"
+msgstr ""
+
+#: src/app/composer/index.tsx:591
+msgid "Add embed for <0>{potential}</0>"
+msgstr ""
+
+#: src/app/composer/index.tsx:488
+#: src/lib/composer/keyboard-accessory.tsx:70
+msgid "Add image"
+msgstr ""
+
+#: src/lib/composer/keyboard-accessory.tsx:53
+msgid "Add image or GIF"
+msgstr ""
+
+#: src/app/(tabs)/(feeds,search,notifications,self)/profile/[author]/lists/[list].tsx:290
+msgid "Add to favourites"
+msgstr ""
+
+#: src/app/_layout.tsx:386
+msgid "Add to List"
+msgstr ""
+
+#: src/app/add-to-list/[handle].tsx:143
+msgid "Add to moderation list"
+msgstr ""
+
+#: src/app/add-to-list/[handle].tsx:134
+msgid "Add to user list"
+msgstr ""
+
+#: src/app/composer/index.tsx:244
+msgid "Adding a description to your image makes Bluesky more accessible to people with disabilities, and helps give context to everyone. We strongly recommend adding alt text to all images."
+msgstr ""
+
+#: src/app/composer/content-warning.tsx:47
+msgid "Adult content"
+msgstr "アダルトコンテンツ"
```

Comment:
This should just be empty I think as it's the English file

Reply:
Whoops yeah good catch